bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -52,7 +52,20 @@
  */
 public abstract class Widget {
 
-	private int zoom;
+	/**
+	 * the zoom level of the widget
+	 * (Warning: This field is platform dependent)
+	 * <p>
+	 * <b>IMPORTANT:</b> This field is <em>not</em> part of the SWT
+	 * public API. It is marked public only so that it can be shared
+	 * within the packages provided by SWT. It is not available on all
+	 * platforms and should never be accessed from application code.
+	 * </p>
+	 *
+	 * @noreference This field is not intended to be referenced by clients.
+	 * @since 3.126
+	 */
+	public int zoom;
```

Comment:
Dito, make it private and provide a (wider) getter
```suggestion
	private int zoom;
```

Reply:
Introducing a public getter/setter will change the API and it is a Win32 specific implementation so we should not do that.